.github/workflows/build.yml

```diff
@@ -9,7 +9,7 @@ jobs:
 
     strategy:
       matrix:
-        java_version: [1.8, 11]
+        java_version: [11]
```

Comment:
We could remove Martix builds, but I like the possibility to add easily other JDKs, thus would keep it